cvp_mesh_planner/include/cvp_mesh_planner/cvp_mesh_planner.h

```diff
@@ -71,23 +71,25 @@ class CVPMeshPlanner : public mbf_mesh_core::MeshPlanner
    * @param message a detailed outcome message
    * @return result outcome code, see the GetPath action definition
    */
-  virtual uint32_t makePlan(const geometry_msgs::PoseStamped& start, const geometry_msgs::PoseStamped& goal,
-                            double tolerance, std::vector<geometry_msgs::PoseStamped>& plan, double& cost,
-                            std::string& message);
+  virtual uint32_t makePlan(const geometry_msgs::msg::PoseStamped& start,
+                            const geometry_msgs::msg::PoseStamped& goal,
+                            double tolerance, 
+                            std::vector<geometry_msgs::msg::PoseStamped>& plan, double& cost,
```

Comment:
you have formatted each variable in a line, but not cost

Reply:
whoops, will fix it!